### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Rdecoster**
first part of clean up.

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -66,6 +67,20 @@ class App extends React.Component {
       });
   }
 
+  updateLike() {
+    alert("I was clicked")
```
Comment (j-scola) [inline client/src/components/App.jsx:71]: is this alert for testing purposes?

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -66,6 +67,20 @@ class App extends React.Component {
...
+    let boolNum = this.state.is_liked;
+    let id = this.state.id;
+    if( boolNum === 0) {
+      this.setState({ is_liked : 1});
+    } else {
+      this.setState({ is_liked : 0});
+    }
+    // var params = [id, boolNum]
+    // axios.get('/updateLike')
+componentDidUpdate
```
Comment (j-scola) [inline client/src/components/App.jsx:81]: i cant tell what componentDidUpdate is doing here

**Review by j-scola — APPROVED** [review]
Looks good! My only general comment is that the indentation was inconsistent and sometimes made things harder to read. I also added some questions.

File: client/src/components/App.jsx (hunk 1 above)
Comment (Rdecoster) [reply client/src/components/App.jsx:71]: yep. Just need to get this branch in with everything else.

File: client/src/components/App.jsx (hunk 2 above)
Comment (Rdecoster) [reply client/src/components/App.jsx:81]: experiment here. for using the modal

**Pull request merged**